docs/gdr-s3-data-sharing.md

```diff
@@ -0,0 +1,110 @@
+
+
+## Step 2 - Prepare an S3 bucket
+
+**TL;DR**  
+Prepare an `ap-southeast-2` S3 bucket that will store the data.
+
+___
+In AWS, you would need to have permissions in order to create/use/modify services. The services needed for this section is full access to S3 bucket and IAM role permission. If you are in the root-account, you do not need to worry about this.
+
+There is a documentation from AWS and some tutorial provided by AWS. You could have a read through and familiarize with the service.
+https://docs.aws.amazon.com/AmazonS3/latest/userguide/GetStartedWithS3.html
+
+
+### Creating the S3 bucket
+
+1. Search on S3 on the search bar, and open the S3 service console.
+2. Click on the `Create bucket` button.
+3. Enter a Bucket name, that is globally unique. Meaning no other people will have this name even on different AWS account. There is some bucket naming rules that need to be followed ([Check it here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html)).
+4. Select AWS Region to `Asia Pacific (Sydney) ap-southeast-2`. Do **NOT** miss this step.
+5. You leave the rest as default and hit the `Create bucket` button.
+6. Your bucket is created!
+
+### Taking the S3 ARN (Amazon Resource Number)
+In AWS, every resource is labeled with an ID called Amazon Resource Number. We need this so that we could refer this resource in the future.
+
+1. Click on the S3 bucket with the name you created on top. (Could use search bucket name if needed)
+2. On the tabs, click on properties. You could see this arn values. Copy this ARN value
+  <br/><img src="./screenshot/s3-arn.png" width="300px"/><br/>
+
+
+## Step 3 - Permission for the AGHA AWS account to send data to your s3 bucket
+
+**TL;DR**  
+Allow AGHA-Service account to upload to S3.  
+
+AWS AGHA role ARN: `arn:aws:iam::602836945884:role/agha-gdr-s3-data-sharing`
+
+___
+
+Before we can send you the data requested to your S3 bucket. You need to give us access to do so.
+
+### Creating Policy on S3 bucket
+We need to create what actions allowed for the permissions.
+1. Click on the S3 bucket with the name you created on top. (Could use search bucket name if needed)
+2. On the tabs, click on `Permissions`. Scroll to see `Bucket policy`, and click on edit.
+  <br/><img src="./screenshot/s3-policy.png" width="500px"/><br/>
+3. Copy the following JSON or append with the following policy. Change the `REPLACE_HERE` below (for `Resource`) value with the bucket ARN above. NOTE: `Resource` has a postfix of `/*`.  
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Principal": {
+              "AWS": "arn:aws:iam::602836945884:role/agha-gdr-s3-data-sharing"
+          },
+          "Action": [
+              "s3:GetObject",
+              "s3:PutObject",
+              "s3:PutObjectAcl",
+              "s3:GetBucketLocation"
+          ],
+          "Resource": "REPLACE_HERE/*"
```

Comment:
Need to check the Resource specifications, I think the GetBucketLocation permission requires a bucket resource ARN, not an object ARN (may need to drop that /* prefix).

Reply:
Yes, agree! Added in local will push it out